.pre-commit-config.yaml

```diff
@@ -30,15 +30,6 @@ repos:
     hooks:
       - id: shellcheck
         args: [--external-sources]
-  - repo: local
```

Comment:
I would remove line 25 (`types_or: ["python", "pyi"]`) of `.pre-commit-config.yaml`.  Prefiltering filetypes can overlook files and this might make sense for slow tools but `ruff` lints the entire CPython codebase in 1/3 of a second with no cache so there is no advantage in putting blinders on it.

Reply:
Sounds good!